src/components/ArticlesList.jsx

```diff
@@ -16,15 +16,18 @@ const ArticlesList = ({ getArticles, setGetArticles }) => {
       return <h2>No Articles</h2>;
     } else {
       return articles.map((article) => (
-        <li key={article.article_id} className="card">
-          <h2>{article.title}</h2>
-
-          <img
-            src={article.article_img_url}
-            alt={`Image off ${article.title}`}
-            className="img"
-          />
-          <h3>{new Date(article.created_at).toLocaleDateString()}</h3>
+        <li className="card" key={article.article_id}>
+          <Link to={`/articles/${article.article_id}`}>
+            <div>
+              <h2>{article.title}</h2>
+              <img
+                src={article.article_img_url}
+                alt={`Image off ${article.title}`}
+                className="img"
+              />
+              <h3>{new Date(article.created_at).toLocaleDateString()}</h3>
+            </div>
+          </Link>
         </li>
       ));
```

Comment:
Having some loading state so the user can get some feedback while the page loads would be beneficial.